Resources/Prototypes/_RMC14/Entities/Structures/Storage/crates.yml

```
@@ -319,6 +319,25 @@
   - type: Icon
     sprite: _RMC14/Structures/Storage/Crates/medical.rsi
 
+- type: entity
+  parent: CMCrateWeldable
+  id: RMCCratePhoron
+  name: phoron crate
+  description: A phoron crate.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Structures/Storage/Crates/phoron.rsi
+    layers:
+    - state: base
+      map: ["enum.StorageVisualLayers.Base"]
+    - state: closed
+      map: ["enum.StorageVisualLayers.Door"]
+    - state: welded
+      visible: false
+      map: ["enum.WeldableLayers.BaseWelded"]
+  - type: Icon
+    sprite: _RMC14/Structures/Storage/Crates/phoron.rsi
+
```

```suggestion
  parent: CMCrateSecure
  id: RMCCratePhoron
  name: phoron crate
  description: A phoron crate.
  components:
  - type: AccessReader
    access: [["CMAccessResearch"],["CMAccessEngineering"]]

```
Drop the new RSI, 13 uses the secure crate, and you'd need access locks sprites, and a place to put the req paperwork, so it's overall better to use `CMCrateSecure` as a parent anyways so you don't need to create anything new.

Make sure to test these changes after though